android/expoview/src/main/java/versioned/host/exp/exponent/modules/universal/notifications/ScopedExpoNotificationCategoriesModule.java

```diff
@@ -0,0 +1,70 @@
+package versioned.host.exp.exponent.modules.universal.notifications;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.os.ResultReceiver;
+
+import org.unimodules.core.Promise;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+
+import expo.modules.notifications.notifications.NotificationSerializer;
+import expo.modules.notifications.notifications.categories.ExpoNotificationCategoriesModule;
+import expo.modules.notifications.notifications.model.NotificationCategory;
+import expo.modules.notifications.notifications.service.BaseNotificationsService;
+import host.exp.exponent.kernel.ExperienceId;
+
+public class ScopedExpoNotificationCategoriesModule extends ExpoNotificationCategoriesModule {
+  private String mExperienceIdString;
+
+  public ScopedExpoNotificationCategoriesModule(Context context, ExperienceId experienceId) {
```

Comment:
It is a good idea to annotate whether method is prepared for Null argument (@Nullable) or it is not (@NotNull). It helps with nullability warnings.

We do not have this everywhere, but if we start adding it, we'll get to the promiseland of null safety sometime :)

Plus, it helps with converting to kotlin ;)


This is rather advise for the future than requirement for this PR, however ;)

Reply:
I haven't used too much of either of those, but agree it's nice to have :) Added it in where useful